invenio_requests/assets/semantic-ui/less/invenio_requests/theme.less

```
@@ -1,3 +1,8 @@
 .ui.segment.loader-container {
   height: 10em;
 }
+
+#request-details-feed-segment {
```

theme.less will be deleted, please move it to site.overrides as:
```suggestion
.shadowless{
box-shadow: none !imporant;
}
```